docs/serving/getting-started-knative-app.md

```diff
@@ -72,31 +88,45 @@ spec:
 ```
 
 If you want to deploy the sample app, leave the config file as-is. If you're
-deploying an image of your own app, update the name of the app and the URL of
+deploying an image of your app, update the name of the app and the URL of
 the image accordingly.
 
-## Deploying your app
-
 From the directory where the new `service.yaml` file was created, apply the
 configuration:
 
 ```bash
 kubectl apply --filename service.yaml
 ```
 
-Now that your service is created, Knative will perform the following steps:
```

Comment:
I think we still want this text here, as these steps still happen for both YAML and `kn` usage.

If you want, you can use the tabbed content to unify the common elements:
```
{{< tabs name="create" default="kn">}}
{{% tab name="kn" %}}
...
{{< /tab >}}
{{% tab name="kubectl" %}}
...
{{< /tab >}}
{{< /tabs >}}
```

I think it's fine to lay one out over the other, though.

Reply:
You mean to use dedicated tabs for the "Creating your Deployment with the Knative CLI" and "Creating your Deployment with YAML" and then only one sectuon "Creating your Deployment" ? 

For this intro case I tend to lay them out one over the other (as is), as the second tab can be easily overlooked and I think its important to prominently present both alternative.

For other sample in the documentation (which I will continue to work on), the tabbed approach makes totally sense.

For now I just added back the list with the steps performed  to the YAML section, too.